Disable hardware acceleration option (#13200)
docker/docker.github.io: desktop/faqs.md

For more information, see [Running Docker Desktop in nested virtualization scenarios](../docker-for-windows/troubleshoot.md#running-docker-desktop-in-nested-virtualization-scenarios).

### I have the problem with displaying some views in Docker Desktop, what should I do?

> Docker Desktop is built on top of Electron which enables hardware accelerated graphics by default, but some graphics cards have issues with hardware acceleration which means Docker Desktop will launch successfully but it will have broken dashboard view.

Reviewer: @mark-dr @trungutt Made some changes. PTAL

> Docker Desktop is built on top of Electron which enables hardware accelerated graphics by default, but some graphics cards have issues with hardware acceleration which means Docker Desktop will launch successfully but it will have broken dashboard view.

Reviewer: Was thinking to link to Electron (https://www.electronjs.org/), but not sure if that's required. Thoughts?

> Docker Desktop is built on top of Electron which enables hardware accelerated graphics by default, but some graphics cards have issues with hardware acceleration which means Docker Desktop will launch successfully but it will have broken dashboard view.

Reviewer 2: I wonder if we even need to mention Electron :thinking:

For more information, see [Running Docker Desktop in nested virtualization scenarios](../docker-for-windows/troubleshoot.md#running-docker-desktop-in-nested-virtualization-scenarios).

### Docker Desktop's UI appears green, distorted, or has visual artifacts. How do I fix this?

> Docker Desktop is built on Electron which enables hardware-accelerated graphics by default. 
> However, some graphics cards may not support hardware acceleration. In such cases, 
> Docker Desktop will launch successfully, but some screens may appear green, distorted, 
> or have some visual artifacts.

Reviewer 2: ```suggestion
Docker Desktop uses hardware-accelerated graphics by default, which may cause problems for some GPUs. In such cases, 
Docker Desktop will launch successfully, but some screens may appear green, distorted, 
or have some visual artifacts.
```

To work around this issue, disable hardware acceleration by 
setting the `DOCKER_DESKTOP_DISABLE_HARDWARE_ACCELERATION` environment 
variable to any value. For example, set `DOCKER_DESKTOP_DISABLE_HARDWARE_ACCELERATION=1` 

> and then restart Docker Desktop.

Reviewer 2: @trungutt We should probably have some instructions about how to do this? E.g. a Powershell snippet for Windows.

For more information, see [Running Docker Desktop in nested virtualization scenarios](../docker-for-windows/troubleshoot.md#running-docker-desktop-in-nested-virtualization-scenarios).

### Docker Desktop's UI appears green, distorted, or has visual artifacts. How do I fix this?

Docker Desktop uses hardware-accelerated graphics by default, which may cause problems for some GPUs. In such cases, 
Docker Desktop will launch successfully, but some screens may appear green, distorted, 
or have some visual artifacts.

> To work around this issue, disable hardware acceleration by turning on `disableHardwareAcceleration` option in `settings.json` for Docker Desktop. This file can be found in

Reviewer 2: ```suggestion
To work around this issue, disable hardware acceleration by creating a `"disableHardwareAcceleration": true` entry in Docker Desktop's settings file. This file can be found at:
```

> - `~/Library/Group Containers/group.com.docker/settings.json` for Mac
> - `C:\Users\{username}\AppData\Roaming\Docker\settings.json` for Windows

Reviewer 2: Just a slight visual improvement:

```suggestion
- **Mac**: `~/Library/Group Containers/group.com.docker/settings.json`
- **Windows**: `C:\Users\[USERNAME]\AppData\Roaming\Docker\settings.json`
```

> and then reopen Docker Desktop's UI.

Reviewer 2: Users probably don't think in terms of UI vs other components. This might be clearer:

```suggestion
Completely close and restart Docker Desktop to apply the changes.
```
